### DISCUSSION TITLE
[求助]番茄被偷吃了！！！

### DISCUSSION THREAD

**Discussion by ZhuYuan-148** [share121/inter-knot#603] [其他]
![1723263915910](https://github.com/user-attachments/assets/28d10b64-a224-4c73-8a57-916a94f1f0fe)
本人是刑侦特勤组的朱鸢，今天给小番茄浇水的时候发现少了两颗！这是答应好给六分街哲先生的礼物，现不知凶手是谁，恳请各位绳匠调查

地点：光辉广场治安局（不是钓鱼执法！）

**Comment by GUILAISHIXV** [comment]
治安局，何时来的？

**Comment by 15976724736** [comment]
钓鱼执法！肯定是钓鱼执法！

**Comment by kaijinti** [comment]
肯定是青衣

**Comment by Tragedysaver1024** [comment]
耍我啊混账，去了之后肯定会被拿下罢（恼）

**Comment by zzz1256** [comment]
前辈偷偷吃掉了！！

**Comment by lllliufeng** [comment]
快去请倒带侦探
